app/_src/installation/docker.md

```diff
@@ -2,17 +2,17 @@
 title: Docker
 ---
 
-To install and run {{site.mesh_product_name}} on Docker execute the following steps:
+To install and run {{site.mesh_product_name}} on Docker, you must do the following:
 
-- [1. Download {{site.mesh_product_name}}](#1-download-kuma)
-- [2. Run {{site.mesh_product_name}}](#2-run-kuma)
-- [3. Use {{site.mesh_product_name}}](#3-use-kuma)
+* Download {{site.mesh_product_name}}
+* Run {{site.mesh_product_name}}
+* Use {{site.mesh_product_name}}
 
 {% tip %}
 The official Docker images are used by default in the [Kubernetes](/docs/{{ page.version }}/installation/kubernetes/) and [OpenShift](/docs/{{ page.version }}/installation/openshift/) distributions.
 {% endtip %}
 
-### 1. Download {{site.mesh_product_name}}
+## Download {{site.mesh_product_name}}
 
 {{site.mesh_product_name}} provides the following Docker images for all of its executables:
 
```

Comment:
Since these installation files were only used in older versions of Kuma, I think Kong Mesh had separate files, so this file isn't shared with the Kong Mesh docs.